content/resources/user/submit-data.md

```diff
@@ -20,6 +20,7 @@ The process of uploading a data project to a Gen3 data commons is simple:
 3. [Map Uploaded Files to a Data File Node](#3-map-uploaded-files-to-a-data-file-node)
 4. [Submit Additional Project Metadata](#4-submit-additional-project-metadata)
 5. [Link Data File Records to their Metadata Records](#5-link-files-to-their-metadata)
+6. [Linking Data from external Data Clouds to Gen3 Data Commons](#6-linking-data-from-external-data-clouds-to-gen3-data-commons)
```

Comment:
I think this should actually be a link in section 2 instead of a whole separate section. Something ~like:

2. Upload Data Files to Object Storage
For local data files, such as spreadsheets, sequencing data (BAM, FASTQ), assay results, images, PDFs, etc., are uploaded to object storage with the gen3-client command-line tool. 
*Note: if your data files are already located in cloud storage, such as an AWS or GCS bucket, please see (this page on how to make these files available in a gen3 data commons.

Reply:
done